### ISSUE TITLE
Handle pom files which already import exec-maven-plugin

### ISSUE THREAD

**Issue by louisq** [louisq/staticguru#26]
POM files that already refer to the exec-maven-plugin cannot have the plugin added to run the staticguru runner. The addition causes a conflict which prevents the build from running. In the mean time we can remove the reference to the exec-maven-plugin and replace it with our own. The current examples which I have seen such as Apache Kylin seem to only be importing in. We would still be doing that, but adding staticguru at the same time

**Comment by louisq** [comment]
Currently only handles the case where one instance is present in the pom file. This will need to be expanded, but it will serve it's purpose for now to unblock the analysis

**Comment by louisq** [comment]
No longer relevant with the completion of #39
